conf/messages

```diff
@@ -504,31 +477,147 @@ technical.technical-error.para1 = Try again. If the problem persists, call us to
+# Decision page
+#********************************************************************************
+pages.decision.title=What do you want to do?
+pages.decision.radio.check=Check current or earlier Marriage Allowance claims
+pages.decision.radio.stop=Stop current Marriage Allowance claim
+pages.decision.error.mandatory.value=Select if you want to check or stop your marriage allowance claim
+
+
+
+#********************************************************************************
+# Make Changes page
+#********************************************************************************
+pages.makeChanges.title=Why do you need to stop your Marriage Allowance?
+pages.makeChanges.radio.divorce=Divorce, end of civil partnership or legally separated
+pages.makeChanges.radio.incomeChanges=Household income changes
+pages.makeChanges.radio.noLongerRequired=Do not want Marriage Allowance anymore
+pages.makeChanges.radio.bereavement=Bereavement
+pages.makeChanges.error.mandatory.value=Select the reason you want to stop your marriage allowance
+
+
+#********************************************************************************
+# Cancel page
+#********************************************************************************
+pages.cancel.title=Cancelling Marriage Allowance
+pages.cancel.paragraph1=We will cancel your Marriage Allowance, but it will remain in place until {0}, the end of the current tax year.
+pages.cancel.paragraph2=Your Personal Allowance will not include any Marriage Allowance from {0}, the start of the new tax year. Your partner will not have to pay back any tax.
+
+
+#********************************************************************************
+# Change of Income page
+#********************************************************************************
+pages.changeOfIncome.title=Change of income
+pages.changeOfIncome.enquiries.link.paragraph=to tell us about a change of income.
+pages.changeOfIncome.paragraph2=You need to tell us if you or your partner’s income changes so we can tell you if:
+pages.changeOfIncome.bullet1=claiming Marriage Allowance will still benefit you as a couple
+pages.changeOfIncome.bullet2=you need to cancel your Marriage Allowance
+
+
+#********************************************************************************
+# Divorce page
+#********************************************************************************
+pages.divorce.title=Date of divorce, end of civil partnership or legal separation
+pages.divorce.paragraph1=You need to go to court to be legally separated. You can still get Marriage Allowance if you are separated, but not legally separated.
+pages.divorce.date.hint=For example, 30 6 2019
+pages.divorce.date.error.mandatory=Enter your date of divorce, end of civil partnership or legal separation
+pages.divorce.date.error.invalid=Enter a real date
+pages.divorce.date.error.non.numeric=Date of your divorce, end of civil partnership or legal separation most only include numbers 0 to 9
```

Comment:
Is it supposed to be "**must** only include"?

Reply:
Yes, good spot :)